Gemfile.lock

```diff
@@ -434,8 +433,8 @@ DEPENDENCIES
   dotenv-rails (= 0.11.1)
   easyzpl!
   execjs
-  factory_girl_rails
-  ffaker (~> 2.8.0)
+  factory_bot_rails
```

Comment:
Hi @steveyken , Looks like we have replaced `factory_girl_rails` with `factory_bot_rails`. Is that ruby 2.5.1 do not support it or its better than factorygirl ? Could you please tell me reason behind it. 

It would be good to know. :)